sda-download/api/sda/sda.go

```diff
@@ -166,6 +167,60 @@ func Download(c *gin.Context) {
 		return
 	}
 
+	// Get query params
+	qStart := c.DefaultQuery("startCoordinate", "0")
```

Comment:
Maybe put detection of coordinates in a function of its own, looking at the query or range header according to the priority we want to handle them in?

Reply:
Same  as https://github.com/neicnordic/sensitive-data-archive/pull/695#discussion_r1522705254, resolving this.